### PR TITLE
fix the alpine repository

### PR DESCRIPTION
**Pull request by thienphamparsley**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ ENV BASE_URL="https://get.helm.sh"
 ENV HELM_3_FILE="helm-v3.4.2-linux-amd64.tar.gz"
 
 RUN apk add --no-cache ca-certificates \
-    --repository http://dl-3.alpinelinux.org/alpine/edge/community/ \
+    --repository http://dl-3.alpinelinux.org/alpine/v3.16/edge/community/ \
```
Comment (mmanzur2) [inline Dockerfile:8]: Doesn't seem like this is a valid path. Check here https://dl-3.alpinelinux.org/alpine/v3.16/

Comment (phunguyen-ph) [reply Dockerfile:8]: Hi @mmanzur2 any suggestions to fix this?

Comment (mmanzur2) [reply Dockerfile:8]: Since this doesn't produce an error, i'm guessing the package library has symlinks. So this is fine for now

**Review by mmanzur2 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (mmanzur2) [reply Dockerfile:8]: @phunguyen-ph I went ahead and merged the pr. lmk if there's any more problems

Comment (phunguyen-ph) [reply Dockerfile:8]: thank you, I am going to run the service again

Comment (phunguyen-ph) [reply Dockerfile:8]: looks good, thanks @mmanzur2

**Pull request merged**